trunk/SUAVE/Input_Output/OpenVSP/vsp_read_fuselage.py

```diff
@@ -0,0 +1,221 @@
+from SUAVE.Components.Fuselages.Fuselage import Fuselage
+import vsp_g as vsp
+import numpy as np
+
+
+## @ingroup Input_Output-OpenVSP
+def vsp_read_fuselage(fuselage_id, units='SI', fineness=True):
+	"""This reads an OpenVSP fuselage geometry and writes it to a SUAVE fuselage format.
+
+	Assumptions:
+	1. OpenVSP fuselage is "conventionally shaped" (generally narrow at nose and tail, wider in center). 
+	2. Fuselage is designed in VSP as it appears in real life. That is, the VSP model does not rely on
+	   superficial elements such as canopies, stacks, or additional fuselages to cover up internal lofting oddities.
+	3. This program will NOT account for multiple geometries comprising the fuselage. For example: a wingbox mounted beneath
+	   is a separate geometry and will NOT be processed.
+	4. Fuselage origin is located at nose. VSP file origin can be located anywhere, preferably at the forward tip
+	   of the vehicle or in front (to make all X-coordinates of vehicle positive).
+	5. Written for OpenVSP 3.16.1
+	
+	Source:
+	N/A
+
+	Inputs:
+	0. Pre-loaded VSP vehicle in memory, via vsp_read.
+	1. VSP 10-digit geom ID for fuselage.
+	2. Units set to 'SI' (default) or 'Imperial'.
+	3. Boolean for whether or not to compute fuselage finenesses (default = True).
+	4. Uses exterior function get_vsp_areas, in SUAVE/trunk/SUAVE/Input_Output/OpenVSP.
+	
+	Outputs:
+	Writes SUAVE fuselage, with these geometries:           (all defaults are SI, but user may specify Imperial)
+
+		Fuselages.Fuselage.			
+			origin                                  [m] in all three dimensions
+			width                                   [m]
+			lengths.
+			  total                                 [m]
+			  nose                                  [m]
+			  tail                                  [m]
+			heights.
+			  maximum                               [m]
+			  at_quarter_length                     [m]
+			  at_three_quarters_length              [m]
+			effective_diameter                      [m]
+			fineness.nose                           [-] ratio of nose section length to fuselage effective diameter
+			fineness.tail                           [-] ratio of tail section length to fuselage effective diameter
+			areas.wetted                            [m^2]
+			tag                                     <string>
+			segment[].   (segments are in ordered container and callable by number)
+			  vsp.shape                               [point,circle,round_rect,general_fuse,fuse_file]
+			  vsp.xsec_id                             <10 digit string>
+			  percent_x_location
+			  percent_z_location
+			  height
+			  width
+			  length
+			  effective_diameter
+			  tag
+			vsp.xsec_num                              <integer of fuselage segment quantity>
+			vsp.xsec_surf_id                          <10 digit string>
+
+	Properties Used:
+	N/A
+	"""  	
+	fuselage = SUAVE.Components.Fuselages.Fuselage()	
+	
+	if units == 'SI':
+		units = Units.meter 
```

Comment:
A name like 'units_factor' would be easier to parse. It would also be better if there were separate variables such as units_type and units_factor so I don't look through the code and wonder why units has changed from a string to a float.